src/AppBundle/Entity/Adherent.php

```diff
@@ -583,7 +583,7 @@ public function getMemberships()
     public function isHost(): bool
     {
         foreach ($this->memberships as $membership) {
-            if ($membership->isHostMember()) {
+            if ($membership->isSupervisor() || $membership->isHostMember()) {
```

Comment:
This method is used in some places for this behavior, perhaps can we remove it but in the meantime we should support it here too.